test-network-function/operator/suite.go

```diff
@@ -83,25 +85,32 @@ var _ = ginkgo.Describe(testSpecName, func() {
 func testOperatorsAreInstalledViaOLM(env *config.TestEnvironment) {
 	testID := identifiers.XformToGinkgoItIdentifier(identifiers.TestOperatorIsInstalledViaOLMIdentifier)
 	ginkgo.It(testID, func() {
+		badOperators := []configsections.Operator{}
 		for _, operatorInTest := range env.OperatorsUnderTest {
 			ginkgo.By(fmt.Sprintf("%s in namespace %s Should have a valid subscription", operatorInTest.SubscriptionName, operatorInTest.Namespace))
-			testOperatorIsInstalledViaOLM(operatorInTest.SubscriptionName, operatorInTest.Namespace)
+			values := make(map[string]interface{})
+			values["SUBSCRIPTION_NAME"] = operatorInTest.SubscriptionName
+			values["SUBSCRIPTION_NAMESPACE"] = operatorInTest.Namespace
+			tester, handlers := utils.NewGenericTesterAndValidate(relativecheckSubscriptionTestPath, relativeSchemaPath, values)
+			context := common.GetContext()
```

Comment:
Maybe we could optimize this a bit by generating the context once, out of the loop? I'm fine with doing it as a follow-up review, though.

Reply:
Yes, we'll review this issue soon for all TCs+TSs.